platform/working-with-vsp/orientation/request-access-to-tools.md

```diff
@@ -1,91 +1,94 @@
 
-#### 4. Configure [the SOCKS proxy](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#configure-the-socks-proxy).
+1. Wait for the GitHub issue to be closed before proceeding.
 
-#### 5. Understand [how to use the SOCKS proxy from inside the VA network and from the internet](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#accessing-socks-proxy-from-va-network).
+1. Configure [the SOCKS proxy](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#configure-the-socks-proxy).
 
-#### 6. [Test and use the SOCKS proxy](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#accessing-socks-proxy-from-the-internet).
+1. Understand [how to use the SOCKS proxy from inside the VA network and from the internet](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#accessing-socks-proxy-from-va-network).
 
-#### 7. Jenkins, Grafana, and Sentry
+1. [Test and use the SOCKS proxy](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#accessing-socks-proxy-from-the-internet).
 
-Jenkins, Grafana, and Sentry have been linked to GitHub for user authentication.
-* The services will not be accessible until the SOCKS proxy is configured and working properly.
-* When logging into these services for the first time click the button `Login with GitHub` or similar.
-  * You will be prompted to link your GitHub account and presented with a permissions dialog.
-  * Allow the service to access your account and view your Organization membership.
+1. Test accessing Jenkins, Grafana, and Sentry through the SOCKS proxy (see below).
+    * When logging into these services for the first time click the button `Login with GitHub` or similar.
+        * You will be prompted to link your GitHub account and presented with a permissions dialog.
+        * Allow the service to access your account and view your Organization membership.
 
-#### 8. For AWS Console access
-##### 8A. When your account has been setup, you will receive a Slack private message with your temporary password and login URL.
-##### 8B. You are required to login and change the temporary password immediately.
-* AWS will prompt you to change your password during the first login.
-* Additionally you are required to setup a virtual MFA device in order to access services in the AWS cloud and programmatically via the CLI.
-  * Follow the walk through for MFA setup [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/backend/aws-account-setup.md).
+1. If you requested AWS Console access:
```

Comment:
Consider making this a subheading instead of a step. It's not really a step in the series. It seems more like additional info for those who requested AWS Console access. Making it a subheading might improve findability/scanability for users who need this info.